tests/sequencer_rotation_test.go

```diff
@@ -6240,3 +6240,2712 @@ func Test_SqcRotation_HisSync_P2P_EVM(t *testing.T) {
+
+	nextProposer, err := dymension.GetNode().GetNextProposerByRollapp(ctx, rollapp1.Config().ChainID, dymensionUserAddr)
+	require.NoError(t, err)
+	require.Equal(t, "sentinel", nextProposer.NextProposerAddr)
+	fmt.Printf("NextProposer: %+v\n", nextProposer)
+
+	currentProposer, err := dymension.GetNode().GetProposerByRollapp(ctx, rollapp1.Config().ChainID, dymensionUserAddr)
+	require.NoError(t, err)
+	require.Equal(t, resp0.Sequencers[0].Address, currentProposer.ProposerAddr)
+	fmt.Printf("CurrentProposer: %+v\n", currentProposer)
+
+	// kick current proposer
+	err = dymension.FullNodes[0].KickProposer(ctx, "sequencer", rollapp1.FullNodes[0].HomeDir())
+	require.NoError(t, err)
+
+	// check client was frozen after kicked
+	clientStatus, err := dymension.GetNode().QueryClientStatus(ctx, "07-tendermint-0")
+	require.NoError(t, err)
+	require.Equal(t, "Frozen", clientStatus.Status)
+
+	rollapp1ValHomeDir := strings.Split(rollapp1.Validators[0].HomeDir(), "/")
+	rollapp1ValFolderName := rollapp1ValHomeDir[len(rollapp1ValHomeDir)-1]
+
+	err = os.RemoveAll(fmt.Sprintf("/tmp/%s/data", rollapp1FolderName))
+	require.NoError(t, err)
+
+	err = os.RemoveAll(fmt.Sprintf("/tmp/%s/data", rollapp1ValFolderName))
+	require.NoError(t, err)
+
+	for i, fullNode := range rollapp1.FullNodes {
+		fmt.Printf("Stopping Full Node %d: %s\n", i, fullNode.Name())
+		err := fullNode.StopContainer(ctx)
+		require.NoError(t, err)
+	}
+
+	err = rollapp1.Validators[0].StopContainer(ctx)
+	require.NoError(t, err)
+
+	err = rollapp1.FullNodes[0].StartContainer(ctx)
+	fmt.Printf("Start Full Node 0: \n")
+	require.NoError(t, err)
+
+	err = rollapp1.FullNodes[1].StartContainer(ctx)
+	fmt.Printf("Start Full Node 1: \n")
+	require.NoError(t, err)
+
+	err = rollapp1.FullNodes[2].StartContainer(ctx)
+	fmt.Printf("Start Full Node 2: \n")
+	require.NoError(t, err)
+
+	err = rollapp1.Validators[0].StartContainer(ctx)
+
+	if err != nil {
+		err = rollapp1.Validators[0].StopContainer(ctx)
+		require.NoError(t, err)
+		err = rollapp1.Validators[0].StartContainer(ctx)
+	}
+
+	// check client was frozen after kicked
+	clientStatus, err = dymension.GetNode().QueryClientStatus(ctx, "07-tendermint-0")
+	require.NoError(t, err)
+	require.Equal(t, "Active", clientStatus.Status)
+
+	time.Sleep(45 * time.Second)
+
+	// Send a normal ibc tx from RA -> Hub
+	transferData = ibc.WalletData{
+		Address: dymensionUserAddr,
+		Denom:   rollapp1.Config().Denom,
+		Amount:  transferAmount,
+	}
+	_, err = rollapp1.SendIBCTransfer(ctx, channel.ChannelID, rollappUserAddr, transferData, ibc.TransferOptions{})
+
+	err = testutil.WaitForBlocks(ctx, 10, dymension, rollapp1)
+	require.NoError(t, err)
+
+	// Check IBC after switch
+	rollappHeight, err = rollapp1.GetNode().Height(ctx)
+	require.NoError(t, err)
+
+	// Assert balance was updated on the hub
+	testutil.AssertBalance(t, ctx, rollapp1, rollappUserAddr, rollapp1.Config().Denom, walletAmount.Sub(transferData.Amount))
+}
```

Comment:
we should check balance of dym's address after ibc transfer to make sure it works